src/plainer.ts

```diff
@@ -166,7 +166,11 @@ export const walker = (
     const seen = seenObjects.get(object);
     if (seen) {
       // short-circuit result if we've seen this object before
-      return seen;
+      return superJson.dedupe
+        ? {
+            transformedValue: null,
+          }
```

Comment:
It'd be nice with some symbol here (and for circular references fwiw) to show in the JSON that we have replaced it

Reply:
Agree. I wonder if there's some other implementation that we can draw inspiration from 🤔 

Apparently Node.js prints `[Circular *1]`:
<img width="266" alt="Screenshot 2023-07-14 at 16 34 28" src="https://github.com/blitz-js/superjson/assets/14912729/aefb6591-bc40-44fe-ae43-a3164b02c106">

Same with Deno. This might even be a v8 thing.

<img width="427" alt="Screenshot 2023-07-14 at 16 44 50" src="https://github.com/blitz-js/superjson/assets/14912729/1a7ecd5e-cead-4e6c-b884-d9699407d036">


What if we did `[Circular]` and `[Pruned]`?